GOVERNANCE.adoc

```diff
@@ -0,0 +1,70 @@
+This document describes the governance rules of the ServiceTalk organization. It is meant to be followed by all
+the projects in the organization.
+
+== Contributions
+Everyone is welcome to contribute to ServiceTalk. Contributions aren't limited to submitting pull requests there are
+many different ways for you to get involved, including answering questions and reporting or triaging bugs. No matter how
+you want to get involved, we ask that you first learn what’s expected of anyone who participates in the project by
+reading the xref:CONTRIBUTING.adoc[Contributor Guidelines].
+
+== Decision-Making and Voting
+ServiceTalk is an open source project, and GitHub serves as the source of truth for the project. Technical disagreements
+may come up, but all community members are always expected to follow the xref:CODE_OF_CONDUCT.adoc[Code of Conduct]. In
+general it is preferred that direction can be mutually agreed upon by community members. If a disagreement cannot be
+resolved independently, the <<Maintainers>> can be called in to vote on the proposal (e.g. issue/PR) in question. An
+independent link:https://github.com/apple/servicetalk/issues[GitHub issues] should be opened prefixed with "[Vote]" and
+reference the proposal in question. The voting period should remain open for at least 2 weeks. Each Maintainer gets one
+vote and the majority will decide the direction. The final decision including the rational for the majority decision
```

Comment:
I wonder if we also need to define that valid votes are: 
 * +1 (yes)
 * 0 (both is fine)
 * -1 (no)

Reply:
I think we need a binary vote as each maintainer is potentially impacted by increased maintenance/work and change in project direction.

Reply:
+1 for binary vote

Reply:
Agreed, a concrete binary opinion is helpful while making a decision. In case "both is fine" a maintainer can skip the voting process.

Reply:
ok fair enough... I just used +1,0,-1 as its a well known concept that is for example used by the projects under the Apache Software Foundation:

https://www.apache.org/foundation/voting.html